### ISSUE TITLE
Page style customization

### ISSUE THREAD

**Issue by maltelau** [djnavarro/jaysire#9]
Would be nice to have a way to include some css. For example, I'd like to include `background-color: black;` in my experiment generating R script rather than having the modify index.html after it's generated.

Perhaps an optional argument for build_experiment would work? For example this would insert some inline css into the `<head>` of `index.html`
```R 
build_experiment(..., style = "<style>.jspsych-display-element {background-color: black;}</style>")
```
or even

```R 
build_experiment(..., style = insert_css(".jspsych-display-element {background-color: black;}"))
```

**Comment by djnavarro** [comment]
yes CSS customisation is on the to do list, but is probably not the top priority at the moment :slightly_smiling_face: given some of the other issues!
